src/System.Private.CoreLib/src/System/Threading/ClrThreadPool.cs

```diff
@@ -46,6 +46,8 @@ private struct CacheLineSeparated
         private static int s_completionCount = 0;
         private static int s_threadAdjustmentInterval;
 
+        private static volatile int s_numRequestedWorkers = 0;
+
         static ClrThreadPool()
```

Comment:
Wondering also because now every access to a member of ClrThreadPool through the singleton instance involves two memory lookups, so there seems to be a tradeoff involved